### ISSUE TITLE
'carpentry' organization

### ISSUE THREAD

**Issue by tracykteal** [carpentries/assessment#153]
Things are organized by 'carpentry' right now. How is this going to work if we have one survey?

Rather than by carpentry should we have 

- surveys (with all of the workshop surveys)
- data (with all of the data)
- reports (with all of the reports)

I know there was just a file structure reorg, so I don't know the background, so this is a point for discussion.

**Comment by kariljordan** [comment]
I don't see us moving to one survey if our curriculum goals differ among Carpentries. The only survey that is common right now is the long-term survey. One thing that can be done eventually is one big report that includes all of the Carpentries (LC,DC,SWC) pre and post data, and that would likely go in the [main folder](https://github.com/carpentries/assessment). I am open to discussing a reorg, but it will take **a lot of time** as I'd have to go in and change all of the file names, then update the dataset file names in all of the reports, and re-render all of the reports. Then all of the links on the websites and blogs will need to be fixed. This is what I had to do when I reorganized the repo in January (I think it was January). Again, if this will ultimately help people find things easier we can map out how it will need to look, but then I'd need to dedicate a day or two to fix all of these things.
